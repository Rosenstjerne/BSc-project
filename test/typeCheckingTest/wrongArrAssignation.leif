var bool[] B;

B = new bool[5];

B[3] = 5;

print(B[3]);