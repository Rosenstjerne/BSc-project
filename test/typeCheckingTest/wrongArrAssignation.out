
Error in phase Type Checking, line 7:
    Trying to assign a value of type 'int' to a varaible of type 'bool' is not possible
